lib/container.js

```diff
@@ -331,6 +331,20 @@ function loadSupportObject(modulePath, supportObjectName) {
   }
   try {
     const obj = require(modulePath);
+
+    if (typeof obj === 'function' && supportObjectName === 'I') {
```

Comment:
```suggestion
    if (typeof obj === 'function') {
```
Do we need to hardcode `I` here?
I think not